cmd/crio/main.go

```diff
@@ -40,8 +41,8 @@ func validateConfig(config *server.Config) error {
 	}
 
 	// This needs to match the read buffer size in conmon
-	if config.LogSizeMax >= 0 && config.LogSizeMax < 8192 {
```

Comment:
Probably does not make sense to set this to BufSize - I can see us adjusting buffer size for various tuning reasons in the future, and we probably don't want to impact minimum log size at the same time